### PR TITLE
Fixes 37235 - Add verify-checksum command for CV versions in hammer

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
1. Add a hammer action for verify-checksum on CV versions
#### What are the testing steps for this pull request?
Run the command below with https://github.com/Katello/katello/pull/10923 checked out.
`hammer -r content-view version verify-checksum --id :id --organization "Default_Organization"`

### CODE REVIEW

**Review by chris1984 — APPROVED** [review]
ACK, code looks fine and works well:

```bash
[vagrant@hammer ~]$ hammer -r content-view version verify-checksum --id 3 --organization "Default_Organization"
[...............................................................................................................] [100%]
```

I would say we add a test but since we are just making an API call to generate a task, i'm fine with skipping it.

**Comment by sjha4** [comment]
Will merge after the katello PR is merged. Thanks @chris1984 !

**Pull request merged**
